setup.cfg

```diff
@@ -1,6 +1,6 @@
 [py2app]
 app = ['bin/meld']
-arch = x86_64
+arch = universal2
```

Comment:
let me know if this works. :)

Reply:
Yep, it works! `universal2` works, `universal` does not.

Reply:
Perfect! Thank you very much for doing this.